mobile/src/main/java/rs/readahead/washington/mobile/views/fragment/reports/ReportsFragment.kt

```diff
@@ -20,12 +20,15 @@ import rs.readahead.washington.mobile.views.fragment.uwazi.SharedLiveData
 @AndroidEntryPoint
 class ReportsFragment :
     BaseBindingFragment<FragmentReportsBinding>(FragmentReportsBinding::inflate) {
-
+    private var initializedView: Boolean = false
```

Comment:
Let's rename it to `isViewInitialized`, do you think we should define this in a `BaseFragment` ?